.gitignore

```diff
@@ -1,8 +1,9 @@
 # Compiled scripts
-Data/Scripts
+
```

Comment:
The `Data/Scripts` folder should contain only `.pex` files (the compiled scripts), so there is no need to add it in the git repository. It is generated during build.

Reply:
Only reason I added the psc files, is some people can't correctly add a mod much less compile the scripts, wasn't sure how active you were on this so I wanted people to be able to download them without having to compile.  feel free to remove this from git.

Reply:
Amending my previous comment, I understand what you are saying now, I uploaded the wrong files to the Scripts directory, duh.

Reply:
Indeed, this directory should only contain compiled files, but I prefer that the git repository only stores source files. The deployment process is responsible for compiling the scripts and packaging them. This should be done automatically by the Github actions (this part is not yet in place), and the git repository should only have the source files manually maintained by developers.

If that's ok with you I'll try to fix this today and merge your PR. Sorry for the hiatus btw. I'm on it.

Reply:
sounds good, thanks